impls/msft-green-ai.yaml

```diff
@@ -0,0 +1,44 @@
+name: mft-green-ai
+description: "https://github.com/Green-Software-Foundation/sci-guide/blob/dev/use-case-submissions/msft-green-ai.md"
+tags:
+  kind: ml
+  complexity: simple
+  category: cloud
+config:
+  funit:
+    - name: job
+      scale-factor: 1 # divide SCI by this value to get SCI per ml job
+  models:
+    - name: sci-c # a model that given e, i and m calculates a carbon value (e * i) + m
```

Comment:
I think we can use a model called sci-o (operational emissions, just energy * grid-ci), still a carbon value but if we have no m then this might be enough.